eiffel-syntax-and-usage/the-meta-object.md

```diff
@@ -18,7 +18,134 @@
 # The Meta Object
 The __meta__ object contains meta-information describing the event: when it was created, where it came from, its type et cetera. The __meta__ object contains the same members regardless of __meta.type__<sup>[1](#footnote1)</sup>, with the caveat that certain members are optional and the tendency to use them may vary with event type.
 
-Even though the meta object is kept universal for all event types, it may change over time as the Eiffel protocol evolves. Due to the [per-event versioning](./versioning.md) of the protocol, this means that different unique combinations of event type, version and meta object version may feature different meta object syntax.
+Even though the meta object is kept universal for all event types (i.e. all events in an edition share the same version of a meta-object, it may change over time as the Eiffel protocol evolves. Due to the [per-event versioning](./versioning.md) of the protocol, this means that different unique combinations of event type, version and meta object version may feature different meta object syntax.
```

Comment:
What do you think of removing line 19 and the footnote and replacing it with the following text?

```suggestion
The __meta__ object contains meta-information describing the event: when it was created, where it came from, its type et cetera. The __meta__ object contains the same members regardless of __meta.type__, with the caveat that certain members are optional and the tendency to use them may vary with event type.

Event types are versioned independently from one another as described in [per-event versioning](./versioning.md) but all events in an edition will have the same version of the meta object. 

To summarize:

1. Any change to meta requires all events to be updated.
1. Consumers should be prepared to receive events of varying meta contents between events. The exception to this are the __meta.type__ and __meta.version__ fields, which are always assumed to be present regardless of event and version.
1. Any schema of a specific version of an event must also include the linked meta object
```